crates/bitwarden-crypto/src/keys/asymmetric_crypto_key.rs

```diff
@@ -29,11 +29,20 @@ impl AsymmetricPublicCryptoKey {
 }
 
 impl AsymmetricEncryptable for AsymmetricPublicCryptoKey {
-    fn to_public_key(&self) -> &RsaPublicKey {
+    fn to_public_rsa_key(&self) -> &RsaPublicKey {
         &self.key
     }
 }
 
+impl FingerprintableKey for AsymmetricPublicCryptoKey {
+    fn fingerprint_parts(&self) -> Vec<Vec<u8>> {
```

Comment:
Thought: DER is a valid canonical encoding; and is implemented for rsa / ed25519 by rustcrypto. However, I'm hesitant to tie ourselves to DER here, because this means that new schemes (ML-DSA, or a hybrid on ML-DSA + ed25519) would not be supported.